tests/integration/cli_test.py

```diff
@@ -5,9 +5,9 @@
 import sys
 from operator import attrgetter
 
-from mock import patch
 from six import StringIO
 
+from .. import mock
```

Comment:
I generally take the approach of importing the module instead of a specific function whenever possible, so that when you read the code it's clear that you're using something external `mock.patch()` and not just a locally defined function `patch()`, but I can't find anything about that in PEP8, so I guess that's just a preference